Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift

```diff
@@ -86,26 +86,26 @@ public struct WorkspaceClientCapabilities: Hashable, Codable, Sendable {
   public struct Symbol: Hashable, Codable, Sendable {
 
     /// Capabilities specific to `SymbolKind`.
-    public struct SymbolKind: Hashable, Codable, Sendable {
+    public struct SymbolKindValueSet: Hashable, Codable, Sendable {
 
       /// The symbol kind values that the client can support.
       ///
       /// If not specified, the client support only the kinds from `File` to `Array` from LSP 1.
       ///
       /// If specified, the client *also* guarantees that it will handle unknown kinds gracefully.
-      public var valueSet: [LanguageServerProtocol.SymbolKind]? = nil
+      public var valueSet: [SymbolKindValueSet]? = nil
 
-      public init(valueSet: [LanguageServerProtocol.SymbolKind]? = nil) {
```

Comment:
These renames don't seem right, these should be the top-level `SymbolKind`, right?

Reply:
You are right.